koku/reporting/provider/aws/models.py

```diff
@@ -274,9 +274,13 @@ class Meta:
         """Meta for OCPUsageTagSummary."""
 
         db_table = "reporting_awstags_summary"
+        unique_together = ("key", "cost_entry_bill")
 
-    key = models.CharField(primary_key=True, max_length=253)
+    id = models.BigAutoField(primary_key=True)
+
+    key = models.CharField(max_length=253)
     values = ArrayField(models.CharField(max_length=253))
+    cost_entry_bill = models.ForeignKey("AWSCostEntryBill", on_delete=models.CASCADE, null=True)
```

Comment:
I would consider removing the null=True. I think we want to require a bill id now so we can guarantee tags get expired the same way as other data.